CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # CHANGELOG
 ## Release 123.0.0 (in development)
 
+### Enhancements
+
+- Bump Kubernetes version to 1.23.0
```

Comment:
I know we weren't doing it before, but wondering, would it make sense to link to the upstream releases when we bump such components?

```suggestion
- Bump Kubernetes version to [1.23.0](https://github.com/kubernetes/kubernetes/releases/tag/v1.23.0)
```